.github/workflows/pr-build.yml

```diff
@@ -9,47 +9,23 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
        # Set fetch-depth: 0 to fetch commit history and tags for use in version calculation
       - name: Check out code
-        uses: actions/checkout@v2.3.4
+        uses: actions/checkout@v3
         with:
           ref: ${{github.event.pull_request.head.ref}}
           repository: ${{github.event.pull_request.head.repo.full_name}}
           fetch-depth: 0
-      
-      - name: create checksum file
-        uses: hypertrace/github-actions/checksum@main
-
-      - name: Cache packages
-        id: cache-packages
-        uses: actions/cache@v2
-        with:
-          path: ~/.gradle
-          key: gradle-packages-${{ runner.os }}-${{ github.job }}-${{ hashFiles('**/checksum.txt') }}
-          restore-keys: |
-            gradle-packages-${{ runner.os }}-${{ github.job }}
-            gradle-packages-${{ runner.os }}
 
       - name: Build with Gradle
         uses: hypertrace/github-actions/gradle@main
         with: 
           args: build
 
-  snyk-scan:
-    runs-on: ubuntu-20.04
+  dependency-check:
```

Comment:
So this is the correct declaration of the job, but if you look at your checks it hasn't run. That's because this is on the pull_request_target workflow - which doesn't take effect until after merge. There's no reason to use that here, so we can either move this into the test workflow (which uses the pull_request trigger, will pick up the change in PR), or what would be even better is removing the `pull_request_target` workflow and scheduling the run anyway, since that's one of the to-do items for all these repos.

An example of that https://github.com/hypertrace/service-framework/pull/70